how-to/install-system-packages.rst

```diff
@@ -13,9 +13,7 @@ version of Ubuntu Linux.
 
 If your project requires a particular system package, you can include them in
 the Docker image, by listing the commands required to install them - typically,
-using ``RUN apt-get`` - in the ``Dockerfile``.
-
-The commands in the ``Dockerfile`` are executed in order, so an appropriate
+using ``RUN apt-get`` - in the ``Dockerfile``. The commands in the ``Dockerfile`` are executed in order, so an appropriate
```

Comment:
Was better previously, please revert this change.

Reply:
Done